agent/xds/listeners.go

```diff
@@ -209,7 +219,7 @@ func (s *ResourceGenerator) listenersFromSnapshotConnectProxy(cfgSnap *proxycfg.
 			}
 
 			filterChain, err := s.makeUpstreamFilterChainForDiscoveryChain(
-				"passthrough",
```

Comment:
Intentional?

Reply:
Actually that should be empty. That field is used for discovery chains IDs to get hooked up as the route name. None of that is relevant for this plain TCP-passthrough.